changelog.yaml

```diff
@@ -14,9 +14,17 @@
 
 #      jiraIssueNumber: <Internal Jira issue number>
 
-- version: "NEXT"
-  date: TBD
-  
+- version: "0.6.0"
+  date: 2020-10-07
+  changes:
+  - type: enhancement
+    impact: minor
+    title: Make jenkins filerunner image configurable in each pipelinerun
+    description: >-
+      You can now configure the jenkins filerunner image to be used in a pipelinerun at 'spec.image'.

```

Comment:
#### Suggestion: 
It would be better to have value for `description` key as a passive sentence - such as - `jenkins-filerunner image could be configurable in Steward-PipelineRun manifest at 'spec.image'`

Reply:
Changed